src/Transports/MassTransit.Azure.ServiceBus.Core/AzureServiceBusTransport/ServiceBusSendTransportContext.cs

```diff
@@ -31,7 +31,7 @@ public class ServiceBusSendTransportContext :
         }
 
         public override string EntityName { get; }
-        public override string ActivitySystem => "service-bus";
+        public override string ActivitySystem => "servicebus";
```

Comment:
Any reason to instrument Azure Service Bus? It has it's own messaging instrumentation and I wonder if we can turn off MassTransit wrapper over it. (as the owner of Azure SDK messaging instrumentations I'm happy to hear any feedback or change things than don't work)

Reply:
Personally, I send my telemetry to Application Insights using an OpenTelemetry collector and the [Azure Monitor exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/azuremonitorexporter) and I find that the result is cleaner when I subscribe to MT activity source:

With `TracerProviderBuilder.AddSource("MassTransit")`:

![MT](https://github.com/MassTransit/MassTransit/assets/147369450/4e9cfff2-3640-42a4-bd56-6b1afabf20c5)

With `TracerProviderBuilder.AddSource("Azure.*")`:

![Azure](https://github.com/MassTransit/MassTransit/assets/147369450/89f94232-857f-4e7c-ad1b-76a69a85389e)
- The send and process activity activities are at the same level (same parent id). I expect the process activity to be under the send one. Probably because the Azure SDK is setting the traceparent in the message's headers before starting the send activity (so the traceparent corresponds to the request POST /orders).
- The names of the spans created by MT are following the [spec](https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#span-name): `<destination name> <operation name>` and are easier to understand

Reply:
> * The names of the spans created by MT are following the [spec](https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#span-name): `<destination name> <operation name>` and are easier to understand

And that's what was done to ensure MassTransit was usuable - it was designed to be useful in tools.